src/ui/pages/config/WidgetGroupings.js

```diff
@@ -35,13 +36,16 @@ const WidgetGroupings = ({
     setOpenGroupings(newOpenGroupings);
   }, [searchFilter, widgetGroupingList]);
 
+  useEffect(() => () => {
+    clearFilter();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
@ichalagashvili I'm thinking aloud, the problem is when someone set the filter, move forward and then come back. I think that instead of at mount, the `clearFilter` should be called on [cleanup](https://reactjs.org/docs/hooks-effect.html#example-using-hooks-1) what do you think?

Reply:
ouch, I didn't see sorry 😅